tests/unit/Api/Service/RevisionSaverTest.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-namespace Wikibase\Api\Test\Service;
-
-use Wikibase\Api\Service\RevisionSaver;
-
-/**
- * @covers Wikibase\Api\Service\RevisionSaver
- */
-class RevisionSaverTest extends \PHPUnit_Framework_TestCase {
```

Comment:
That's a lot of lines of test code to test the constructor which has no logic anyway. Also found some other very dubious tests, though those did not break, so I left them alone.